docs/api/browser-window.md

```diff
@@ -385,6 +385,8 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
       visible to users.
     * `spellcheck` Boolean (optional) - Whether to enable the builtin spellchecker.
       Default is `true`.
+    * `enableWebSQL` Boolean (optional) - Whether to enable the [WebSQL api](https://www.w3.org/TR/webdatabase/).
+      Default is `true`.
```

Comment:
Do we plan to set the default to false in a followup?

Reply:
Thats up for discussion, I didn't know if we wanted to keep it enabled by default upto a certain electron version and then disable it ?